test/verify/check-kdump

```diff
@@ -486,5 +514,162 @@ class TestKdumpNFS(KdumpHelpers):
             raise
 
 
+@testlib.skipOstree("kexec-tools not installed")
+@testlib.onlyImage("Only Fedora has ansible on the image", "fedora-*")
+@testlib.timeout(900)
+class TestKdumpNFSAnsible(KdumpHelpers, packagelib.PackageCase):
+    provision = {
+        "ansible_machine": {"address": "10.111.113.1/24", "memory_mb": 1024, "capture_console": True},
+        "nfs": {"image": TEST_OS_DEFAULT, "address": "10.111.113.2/24", "memory_mb": 512},
+        "cockpit": {"memory_mb": 512},
+    }
+
+    def testBasic(self):
+        m = self.machine
+        b = self.browser
+
+        self.enableNFSServer()
+
+        m2 = self.machines["cockpit"]
+        m2.start_cockpit()
+
+        b2 = self.new_browser(m2)
+        b2.login_and_go("/kdump")
+
+        # Create a fake repo so dnf does not hang trying to refresh the metadata cache
+        # which requires internet access.
+        self.createPackage("kexec-tools", "1", "1", content={"/kexec-tools": ""})
+        self.enableRepo()
+
+        self.login_and_go("/kdump")
+        self.assertActive(active=False)
+
+        # switch to NFS
+        b2.click("#kdump-change-target")
+        b2.wait_visible("#kdump-settings-dialog")
+        b2.set_val("#kdump-settings-location", "nfs")
+        b2.set_input_text("#kdump-settings-nfs-server", "10.111.113.2")
+        b2.set_input_text("#kdump-settings-nfs-export", "/srv/kdump")
+        b2.click(f"#kdump-settings-dialog button{self.primary_btn_class}")
+        b2.wait_not_present("#kdump-settings-dialog")
+
+        # This requires a reboot
+        self.run_ansible_playbook(b2, allow_failure=True)
+        conf = m.execute("cat /etc/kdump.conf")
+        self.assertIn("\n# Ansible managed\n", conf)
+        self.assertIn("\nnfs 10.111.113.2:/srv/kdump\n", conf)
+
+        m.reboot()
+        self.login_and_go("/kdump")
+
+        # Verify that kdump runs and crashkernel is configured
+        m.execute("until systemctl is-active kdump; do sleep 1; done")
+        b.wait_in_text("#app", "Service is running")
+        b.wait_in_text("#app", "192 MiB")
+        self.assertActive(active=True)
+
+        try:
+            self.crashKernel("copied through NFS to 10.111.113.2:/srv/kdump/var/crash")
+
+            # dump is done during boot, so should exist now
+            self.assertIn("Kdump compressed dump",
+                          self.machines["nfs"].execute("file /srv/kdump/var/crash/10.111.113.1*/vmcore"))
+        except (testlib.Error, AssertionError):
+            self.machine.print_console_log()
+            raise
+
+
+@testlib.skipOstree("kexec-tools not installed")
+@testlib.onlyImage("Only Fedora has ansible on the image", "fedora-*")
+@testlib.timeout(900)
+class TestKdumpAnsible(KdumpHelpers, packagelib.PackageCase):
+    provision = {
+        "ansible_machine": {},
```

Comment:
Please try to reduce RAM

Reply:
1024 is a tad better, but 512 or 768 ought to suffice? (it may be that crashkernel reserves less RAM for that, of course)

Reply:
With 768 you'll get:
```
Nov 23 11:21:41 fedora-39-127-0-0-2-2201 systemd[1]: Starting kdump.service - Crash recovery kernel arming...
Nov 23 11:21:41 fedora-39-127-0-0-2-2201 kdumpctl[899]: kdump: No memory reserved for crash kernel
```

So that's why it's 1024. Note that's less then `TestKdump` as the default VM size is ~ 1100? And 1024 is the same as TestKdumpNFS

Reply:
I can offer 900 this seems to work, at least locally.

Reply:
Thanks for checking! This test just allocates two VMs, so 1024 is fine -- let's not push it too close to the "brittle" line.